tools/Scripts/CopyNativeDlls.bat

```diff
@@ -0,0 +1,4 @@
+@ECHO OFF
+SETLOCAL
+SET PowerShellScriptPath=%~dpn0.ps1

```

Comment:
>PowerShellScriptPath=%~dpn0.ps1 [](start = 4, length = 31)

Nit: I think we should consider naming these "CopyPluginsToUnity.bat" / "CopyPluginsToUnity.ps1", or something that contains Unity in the name.